cocos/core/director.ts

```diff
@@ -290,7 +291,8 @@ export class Director extends EventTarget {
         this._scene = null;
 
         // FPS
-        this._totalFrames = 0;
+        this._poolHandle = DirectorPool.alloc();
+        DirectorPool.set(this._poolHandle, DirectorView.TOTAL_FRAMES, 0);
```

Comment:
Why director needs to be stored into Pool ? If total frames is required in renderer, why don't the renderer count by itself

Reply:
Ok, can calculate it in native. It is fixed.